doc/nrf/known_issues.rst

```diff
@@ -1139,6 +1237,26 @@ Assert on writing to flash
 Timeout without sending packet
   A directed advertiser may time out without sending a packet on air.
 
+.. rst-class:: v0-2-0 v0-1-0
+
+DRGN: HCI Read Local Supported Commands command do not indicate support for HCI LE Set Privacy Mode command.
+  HCI Read Local Supported Commands command do not indicate support for HCI LE Set Privacy Mode command.
+
+.. rst-class:: v0-2-0 v0-1-0
+
+DRGN: An ASSERT occurs when setting advertising data after HCI Reset without setting advertising parameters.
+  An ASSERT occurs when setting advertising data after HCI Reset without setting advertising parameters.
+
+.. rst-class:: v0-2-0 v0-1-0
+
+DRGN: An ASSERT may occur when writing to flash.
+  An ASSERT may occur when writing to flash.
+
+.. rst-class:: v0-2-0 v0-1-0
+
+DRGN: Directed advertisers may time out without sending a packet on air.
+  Directed advertisers may time out without sending a packet on air.
```

Comment:
Is there something else that could be used for the second lines? The duplication looks unprofessional.

Reply:
Unfortunately I could not find jiras for those issues, so there is no extra information. But I can try to rephrase a bit.

Reply:
I would appreciate it.